src/Attributes/RegisterAlgoliaIndexAttribute.cs

```diff
@@ -26,6 +26,7 @@ public Type Type
         public string IndexName
         {
             get;
+            set;
```

Comment:
This should be marked as internal. For consumers it's a mandatory field (forced by ctor parameter) and shouldn't be changable.
+ It generates weird intellisense for attribute, the field is available in ctor and  again as an optional property

Reply:
Marked as internal in https://github.com/Kentico/xperience-algolia/pull/24/commits/888a501ff25850ecc57517fdde90dfed3deed90d